mixer/test/client/env/envoy.go

```diff
@@ -42,7 +42,7 @@ func (s *TestSetup) NewEnvoy() (*Envoy, error) {
 
 	debugLevel := os.Getenv("ENVOY_DEBUG")
 	if len(debugLevel) == 0 {
-		debugLevel = "info"
+		debugLevel = "trace"
```

Comment:
? this is going to spew the logs with too much info making it unusable. Please set it to info

Reply:
it only shows up in failures where it's useful. forgot to remove it while debugging.